app/assets/javascripts/admin/variant_overrides/controllers/variant_overrides_controller.js.coffee

```diff
@@ -13,6 +13,10 @@ angular.module("admin.variantOverrides").controller "AdminVariantOverridesCtrl",
   $scope.RequestMonitor = RequestMonitor
   $scope.selectView = Views.selectView
   $scope.currentView = -> Views.currentView
+  $scope.onDemandOptions = [
```

Comment:
Why didn't you include the third option in here?

Reply:
I wasn't able to get this to work at first, but I think this was only in the initial version when I was using string counterparts of `null`, `true`, and `false`. Angular automatically added a blank option so I just *renamed* the label, but it turns out it wouldn't add this if there is a match.

Fixed already: https://github.com/openfoodfoundation/openfoodnetwork/pull/3104/commits/517a7c85f257d16aadc2c621561dbb7c721d3c6e

Thanks, @mkllnk! :slightly_smiling_face: